src/carts/Dockerfile

```diff
@@ -1,116 +1,44 @@
 ##### Application Metadata
 ARG APPLICATION_NAME="carts"
 ARG VERSION="0.0.1-SNAPSHOT"
-##### Oracle Instant Client version
-ARG JDBC_DRIVER_VERSION="19.3"
 
 # ------------
-# Stage 1 : Download Oracle JDBC drivers. 
+# Stage 1 : Build the application using Maven
 # 
-# 
-FROM oraclelinux:7-slim as oracleinstantclientjdbc
-
-ARG JDBC_DRIVER_VERSION
-
-RUN yum update -y && \
-  yum -y install oracle-release-el7 && \
-  yum-config-manager --enable ol7_oracle_instantclient && \
-  yum -y install oracle-instantclient${JDBC_DRIVER_VERSION}-jdbc && \
-  yum clean all && \
-  rm -rf /var/cache/yum
-
-#
-# ------------
-
-# ------------
-# Stage 2 : Setting up the build environment with maven and gradle
-# 
-# 
-FROM maven:3.6-jdk-8 as buildenv
-
-ARG JDBC_DRIVER_VERSION
-ARG JDBC_GROUP_ID="com.oracle.jdbc"
-ARG JDBC_DRIVER_ARTIFACT_ID="ojdbc8"
-ARG GRADLE_VERSION=5.5.1
-
-ENV GRADLE_HOME /opt/gradle
+FROM maven:3.6-jdk-8 as appbuild
 
-COPY --from=oracleinstantclientjdbc /usr/lib/oracle/${JDBC_DRIVER_VERSION}/client64/lib/ojdbc8.jar /usr/lib/oracle/${JDBC_DRIVER_VERSION}/client64/lib/ojdbc8.jar
+# Get SODA Java
+ADD https://github.com/oracle/soda-for-java/releases/download/v1.1.0/orajsoda-1.1.0.jar /tmp/orajsoda-1.1.0.jar 
```

Comment:
version should be a variable set on the beginning of the Dockerfile.

Reply:
Fixed.